docs/lang/articles/advanced/layout.md

```diff
@@ -3,58 +3,37 @@ sidebar_position: 2
 ---
 
 # Fields (advanced)
+Morden processor cores are orders of magnitude faster than their equipped memory systems. In quite a few scenarios, programs are bound by the speed of memory rather than cores. In order to shrink this speed gap, the multi-level cache system and high-bandwidth multi-channel memories are built into computer architectures.
 
-This section introduces some advanced features of Taichi fields.
-Make sure you have gone through [Fields](../basic/field).
+There arises two questions in making performance optimization efforts: 1) how to organize a faster data layout and 2) how to manage memory occupancy. In this article, we discuss in the scope of Taichi field. A better understanding of the mechanism under the hood is essential to write blazing fast Taichi programs.
```

Comment:
Suggest the following ;/

After familiar yourself with the basics of Taichi [Fields](../basic/field), this article helps you one step further by explaining the underlying memory layout that is essential to write high-performance Taichi programs. In particular, we present how to organize a faster data layout and how to manage memory occupancy.

Reply:
The revision is nice! Resolved by latest commit.